x/bank/keeper/keeper.go

```diff
@@ -439,6 +439,10 @@ func (k BaseKeeper) setSupply(ctx context.Context, coin sdk.Coin) {
 func (k BaseKeeper) trackDelegation(ctx context.Context, addr sdk.AccAddress, balance, amt sdk.Coins) error {
 	acc := k.ak.GetAccount(ctx, addr)
```

Comment:
I'm wondering if we should provide a way for GetAccount to work for x/accounts. 

I think the way i'd handle this is:
- GetAccount
- try on x/auth state
- If not exist:
   - check on x/accounts
   - fetch account type from x/accounts
       - if account type == "base_account" (the one living in default/base)
           - then convert it into BaseAccount struct
       - else: return it as a private implementation.

Reply:
but for the sake of pushing this to unblock e2e test for auditing, we can move this to another PR